thrift/binary_protocol.go

```diff
@@ -86,8 +86,7 @@ func (p *TBinaryProtocol) WriteMessageBegin(name string, typeId TMessageType, se
 		if e != nil {
```

Comment:
this is vendored code (see https://github.com/jaegertracing/jaeger-client-go/blob/master/thrift/README.md), we don't want to change it

Reply:
Sure! Thanks for the clarification.

Reply:
Since most of the changes are in the thrift vendored package, I will close this pull request.